### PR TITLE
{vis}[GCCcore/8.2.0] FFmpeg v4.1.3, x264 v20190413, LAME v3.100, Yasm v1.3.0, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.1.3-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '4.1.3'
+
+homepage = 'https://www.ffmpeg.org/'
+description = """A complete, cross-platform solution to record, convert and stream audio and video."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
```
Comment (boegel) [inline easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.1.3-GCCcore-8.2.0.eb]: @Micket Don't we pay a price performance-wise by not compiling `FFmpeg` with Intel compilers?

Comment (Micket) [reply easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.1.3-GCCcore-8.2.0.eb]: @boegel Not really. x264/x265 were already at GCCcore level. And for all the builtin stuff FFmpeg does a lot of manual assembly code in all the performance hotspots.
Secondly, I somehow doubt HPC time spent encoding movies are going to be insignificant. 
Thirdly, i wouldn't think that icc is necessarily really that amazing at compiling software like this.

But, just to be sure, I actually just tested building FFmpeg and FFmpeg+x264 at iccifort level, and benchmarked converting and reencoding some full HD movie trailer i grabbed online. iccifort versions were slightly worse in all tests i made (by ~5%)

Comment (boegel) [reply easybuild/easyconfigs/f/FFmpeg/FFmpeg-4.1.3-GCCcore-8.2.0.eb]: Excellent, thanks for diving into this!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node3109.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/3a0726fab7ad6bff04d7cec78d201819 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2657.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4829db2c1692d94c6447e42cd5fa7683 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
node2023.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4c07c766e3fcf5ea99a0b23b5db934bc for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Micket!

**Pull request merged**